lib/src/lint_action.clj

```diff
@@ -59,12 +59,14 @@
        (apply concat)
        (cstr/join "/")))
 
-(defn- run-clj-kondo [dir relative-files relative-dir]
-  (let [kondo-result
-        (sh "sh" "-c"
-            (str "cd " dir ";"
+(defn- run-clj-kondo [dir relative-files relative-dir options]
+  (let [options (if options options {})
+        command (str "cd " dir ";"
                  "/usr/local/bin/clj-kondo " "--lint "
-                 (cstr/join \: relative-files)))
+                 (cstr/join \: relative-files)
+                 " --config " (pr-str (pr-str options)))
+        kondo-result
+        (sh "sh" "-c" command)
```

Comment:
You can leave it as is, but you can specify the working directory for `sh` with the [`:dir`](https://clojuredocs.org/clojure.java.shell/sh) option. That way, you don’t have to use `sh -c` and it allows you to remove those nested calls to `pr-str`, like:

```clojure
(let [...
      command ["/usr/local/bin/clj-kondo"
               "--lint" (cstr/join \: relative-files)
               "--config" (pr-str options)]
      kondo-result (apply sh (concat command [:dir dir]))
      ...]
  ...)
```